### PR TITLE
Email confirmation update

### PR DESCRIPTION
**Pull request by ansahdavid**

### CODE REVIEW

File: app/services/DateService.scala [hunk 1]
```diff
@@ -42,9 +42,18 @@ class DateService @Inject()(clock: Clock) {
...
+  }
+
+  def lastDayOfMonthAfterCalendarQuarter: LocalDate = {
+    startOfNextPeriod.withDayOfMonth(startOfNextPeriod.lengthOfMonth)
+  }
+
   def earliestSaleAllowed: LocalDate = {
     val quarterStartMonths = Set(JANUARY, APRIL, JULY, OCTOBER)
     val today = LocalDate.now(clock)
+    println("Today: " + today) // 2022-02-06   2021-07-14
```
Comment (TheNickWilson) [inline app/services/DateService.scala:56]: Could we remove this please?

File: app/services/DateService.scala [hunk 2]
```diff
@@ -42,9 +42,18 @@ class DateService @Inject()(clock: Clock) {
     }
   }
 
+  def lastDayOfCalendarQuarter: LocalDate = {
```
Comment (TheNickWilson) [inline app/services/DateService.scala:45]: Could we consolidate the naming in this class? We sometimes refer to "Period" and sometimes to "CalendarQuarter" - we should pick one and use it consistently (I don't mind which!)

File: app/services/DateService.scala (hunk 1 above)
Comment (ansahdavid) [reply app/services/DateService.scala]: Done

File: app/services/DateService.scala (hunk 2 above)
Comment (ansahdavid) [reply app/services/DateService.scala:45]: Yep agreed! Have now replaced "Period" with "Quarter"

**Review by susan-badel — APPROVED** [review]

**Pull request merged**